src/main/java/com/thirdcc/webapp/repository/EventAttendeeRepository.java

```diff
@@ -12,4 +17,5 @@
 @Repository
 public interface EventAttendeeRepository extends JpaRepository<EventAttendee, Long> {
 
+    Optional<Event> findOneByIdAndStatusIn(Long id, Set<EventStatus> eventStatus);
```

Comment:
do you use this method in your code?
Why did you write it?

Reply:
I think I put it in the wrong place, supposedly just in the EventRepository.